SH4RE/SH4RE/Search/FilterSheetView.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  FilterSheetView.swift
+//  SH4RE
+//
+//  Created by Americo on 2023-01-19.
+//
+
+import SwiftUI
+import Combine
+
+struct LocationEntryField: View {
+    @Binding var location: String
+    var body: some View {
+        HStack {
+            TextField("Location", text: $location)
+                .textFieldStyle(
+                    locationInputStyle(
+                        button: Button(action:{
+                            // Currently no action, have to set up location
+                            // This could be a staring point https://www.youtube.com/watch?v=cOD1l2lv2Jw&ab_channel=azamsharp
+                            // Additionally, check out the file "CurrentLocationButton"
+                        }, label:{
+                            Image(systemName: "scope")
+                        })
+                    )
+                )
+        }
+    }
+}
+
+struct FilterSheetView: View {
+    
+    var dropDownList = ["Film & Photography", "Audio Visual Equipment", "Projectors & Screens", "Drones", "DJ Equipment", "Transport", "Storage", "Electronics", "Party & Events", "Sports", "Musical Instruments", "Home, Office & Garden", "Holiday & Travel", "Clothing"]
+    @State private var dropDownSelection: String = ""
+
+    @State private var location: String = ""
+    @State private var minPrice: String = ""
+    @State private var maxPrice: String = ""
+    @State private var maxDistance: String = ""
+    
+    @Binding var showingFilterSheet: Bool
+    
+    @State private var minRating = 0.0
+
+    fileprivate func NumericTextField(label: String, textEntry: Binding<String>) -> some View {
+        return TextField(label, text: textEntry)
+            .keyboardType(.numberPad)
+            .textFieldStyle(textInputStyle())
+            .onReceive(Just(textEntry.wrappedValue)) { newValue in
+                let filtered = newValue.filter { "0123456789".contains($0) }
+                if filtered != newValue {
+                    textEntry.wrappedValue = filtered
+                }
+            }
+    }
+    
+    var body: some View {
+        ScrollView {
+            VStack (alignment: .leading){
+                Group {
+                    HStack {
+                        Text("Filter")
+                            .font(.title)
```

Comment:
using this vs the system font... did we decide?

Reply:
personally i like this way better

Reply:
i think this is stull the built in title font, not our custom version. we are able to redefine what .title means tho I believe...

Reply:
yeah we arent using any custom fonts, didnt put any effort into figuring that out. instead tho ig we just have a mix of the .title/.caption etc and .system(.font: 14) etc.